distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Redshift.enso

```diff
@@ -33,39 +35,45 @@ type Redshift
         if Driver.isRegistered.not then Driver.register
 
         properties = options.merge self.jdbc_properties
+        java_props = Properties.new
+        properties.each pair->
+            java_props.setProperty pair.first pair.second
         Connection.create_jdbc_connection self.jdbc_url properties self.dialect
 
     ## Provides the jdbc url for the connection.
     jdbc_url : Text
     jdbc_url =
         prefix = case self.credentials of
-            AWS_Profile _ -> 'jdbc:redshift:iam://'
+            AWS_Profile _ _ -> 'jdbc:redshift:iam://'
+            AWS_Key _ _ _ -> 'jdbc:redshift:iam://'
             _ -> 'jdbc:redshift://'
         prefix + self.host + ':' + self.port.to_text + (if self.schema == '' then '' else '/' + self.schema)
 
     ## Provides the properties for the connection.
     jdbc_properties : [Pair Text Text]
     jdbc_properties =
         credentials = case self.credentials of
-            Nothing -> PostgreSQL.read_pgpass self.host self.port self.schema
-            AWS_Profile profile -> if profile == '' then [] else [Pair 'profile' profile]
+            Nothing -> PostgreSQL.PostgreSQL.read_pgpass self.host self.port self.schema
+            AWS_Profile db_user profile ->
+                [Pair 'user' db_user] + (if profile == '' then [] else [Pair 'profile' profile])
+            AWS_Key db_user access_key secret_access_key ->
+                [Pair 'user' db_user, Pair 'AccessKeyID' access_key, Pair 'SecretAccessKey' secret_access_key]
             Credentials username password ->
                 [Pair 'user' username, Pair 'password' password]
 
-        ssl_properties = PostgreSQL.ssl_mode_to_jdbc_properties self.use_ssl
+        ## ssl_properties = PostgreSQL.PostgreSQL.ssl_mode_to_jdbc_properties self.use_ssl
+        ssl_properties = []
```

Comment:
Why is this commented out? I think at least some explanation for why this is kept as a comment should be added.